index.html

```diff
@@ -3635,6 +3643,42 @@ <h4>Protocol Binding based on HTTP</h4>
                     <code>Form</code> with operation type <code>invokeaction</code>
                 </td>
             </tr>
+            <tr class="rfc2119-default-assertion" id="td-default-http-method_readall">
```

Comment:
I would it keep more simple:  in the first row in the table I would extend in the last column "Form with operation type readproperty, readallproperties, or readmultipleproperties"

Reply:
I agree, you should see the change in the new commit